dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/hadoop/HadoopMetadataConnector.java

```diff
@@ -82,13 +100,40 @@ public void addTasksTo(@Nonnull List<? super Task<?>> out, @Nonnull ConnectorArg
         .map(scriptName -> new BashTask(scriptName, HadoopScripts.extract(scriptName + ".sh")))
         .forEach(out::add);
     addTasksForSingleLineScripts(out);
+    generateServiceScripts(out);
   }
 
   private void addTasksForSingleLineScripts(List<? super Task<?>> out) {
     HadoopScripts.extractSingleLineScripts()
         .forEach((scriptName, scriptFile) -> out.add(new BashTask(scriptName, scriptFile)));
   }
 
+  private void generateServiceScripts(List<? super Task<?>> out) {
+    SERVICE_NAMES.stream()
+        .flatMap(
+            serviceName ->
+                Stream.of(
+                    generateBashTask(
```

Comment:
Optional: The `String`s here could also be converted to a class that would represent a service and contain the methods `toServiceStatus` and `toSystemCtlStatus`.

Reply:
The code is tiny - 6 lines of code. I don't predict any further processing of these scripts in Java, so IMHO investing more time in representing the process as Java entities instead of Strings will not help in the future.